### ISSUE TITLE
Navigation View with Versions, Downloads, Search, etc. Not Working

### ISSUE THREAD

**Issue by MonamiB** [rtfd/readthedocs.org#1182]
RTD on localhost is not rendering the navigation view with Versions, Downloads, Search, and other links. Here's the screenshots of ideal view vs. actual. Are there any known fixes?

![rtd_versionsview_ideal](https://cloud.githubusercontent.com/assets/11224494/6572989/b4d9d4da-c73c-11e4-8467-5019899d4ed1.png)
![rtd_versionnav_err](https://cloud.githubusercontent.com/assets/11224494/6572990/b5f2ae14-c73c-11e4-9baa-dbb2575d4cd9.png)

**Comment by agjohnson** [comment]
I haven't encountered this and there isn't enough here to reproduce this issue. I'd try inspecting the javascript that updates the menu -- readthedocs-doc-embed.js is the piece that handles this.

**Comment by MonamiB** [comment]
Essentially, I have built multiple versions of the same reST source using Sphinx. I have installed RTD on ubuntu OS and am running a local instance at http://127.0.0.1:8000/
I can view all the versions individually through the versions page at http://127.0.0.1:8000/projects/test/versions/  The PDF and ePub are generating fine, no errors. Noticed an error now. 
----
[16/Mar/2015 00:46:19] "GET /media/javascript/readthedocs-doc-embed.js HTTP/1.1" 404 1752
----
What does this indicate? What should I be fixing/changing?
